### PR TITLE
Project Completion

### PR DESCRIPTION
**Pull request by rebaz36**
### We've worked individually and collaboratively as needed based on the tasks at hand to finalize our project.

- Teammate 1: @JonahKayizzi 
- Teammate 2: @eng-mohamed-eldeeb 
- Teammate 3: @rebaz36 

#### we'd love to receive any constructive feedback, 👨🏼‍💻

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,181 @@
...
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [updated upon project completion.]()
+
```
Comment (alexander16108) [inline README.md]: The following basic requirements are required for your project to proceed to the next phase of your capstone 😄 

-  you should record a video presenting the features of the project.
- The maximum length of the video is 5 minutes.
- The video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
-  All the members in the video should appear in the video and talk about the project.
- The video contains a highlight about a piece of the code the student finds interesting or is proud of making.

**Review by alexander16108 — CHANGES_REQUESTED** [review]
Hi Team 👋🏾 , 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!



## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md (hunk 1 above)
Comment (rebaz36) [reply README.md]: Thanks for letting us know @alexander16108 we were so focused on the completion of the project that we totally overlooked this requirement, we'll add it in no time

**Review by Ghiftee — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️♻️
Hi team 👋🏽 , 

Great work on making the changes requested by a previous reviewer 👏 
To highlight 👍🏽 
✔️ You added a presentation video
✔️ You spoke a code you are proud of
✔️ All team members appeared in the video
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Required Changes ♻️
- [ ] According to the requirements shown below, your video should not exceed 5 minutes. It is currently over 7 minutes long, kindly shorten it.
Screenshot 2023-01-12 at 13 45 58

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by JonahKayizzi** [comment]
A new video that is less than 5 minutes has been shared in the README

**Review by BenMukebo — APPROVED** [review]
# Status: Approved  :champagne:  ✔️:confetti_ball: 

## Hi @TEAM, 

### **Congratulations! 🎉 Project Approved ✅**

Good job so far. Well Done.👍🏻 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 


### To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Git is followed. :heavy_check_mark:
- Good presentation video. ✔️ 
- Each class from the [UML class diagram](https://github.com/microverseinc/curriculum-ruby/blob/main/group-capstone/images/catalog_of_my_things.png) was created in a separate .rb file 👍🏻 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers :clinking_glasses:  and Happy coding!!! :dancers:💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**